phpunit.xml

```diff
@@ -1,12 +1,23 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/7.5/phpunit.xsd"
+         xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/8.5/phpunit.xsd"
          bootstrap="vendor/autoload.php"
          colors="true"
+         executionOrder="depends,defects"
```

Comment:
I could not find any documentation on what the eight possible values actually mean.

Reply:
depends: If test A depends on test B (as marked by an annotation), execute B before A

defects: Run those tests first that have failed during the previous run